src/auth-service/routes/v2/health.routes.js

```
@@ -0,0 +1,140 @@
+        message: "Internal server error in health check",
+        errors: { message: "Failed to process health check" },
+      });
+    }
+  });
+
+  // Message broker health check - protected with authentication and rate limiting
+  router.get(
+    "/message-broker",
+    (req, res, next) => {
+      try {
+        if (healthValidations && healthValidations.rateLimit) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)